functions/src/project/delete-project.ts

```diff
@@ -31,6 +32,8 @@ async function deleteProjectRepositories(projectUid: string, project: ProjectMod
               t.update(repoRef, { projects: repoData.projects });
             });
         });
+
+        await deleteRepoBuilds(repositoryUid);
```

Comment:
The above delete is done as a `transaction`, it would be better to be part of the `transaction`

Reply:
Moved this function call inside transaction and below `t.update(repoRef, { projects: repoData.projects });`